BlueMap/Chat/src/main/java/codes/antti/auth/bluemap/chat/BlueMapChatPlugin.java

```diff
@@ -179,6 +209,24 @@ public void onChatMessage(AsyncPlayerChatEvent event) {
 		forEachSession(sse -> sse.send(message));
 	}
 
+	@EventHandler(priority = EventPriority.MONITOR)
+	public void onServerCommand(ServerCommandEvent event) {
+		// Check if the command is a 'say' command
```

Comment:
Do you want no comments for everything? Including JS files? Personally I like to put comments in so that other people can easily read the code and so I can understand things if coming back to the code after a while.

Reply:
I think I'd prefer that as currently it feels awkward because my old code has basicly none